src/controller/users/authorsGet.ts

```diff
@@ -0,0 +1,15 @@
+import { pool } from '../../DB/dbConfig';
+import { Request, Response } from 'express';
+
+export async function authorsGet(_req: Request, res: Response): Promise<void> {
+	try {
+		const authors = await pool.query(`SELECT * FROM users WHERE is_author = $1`, ['true']);
```

Comment:
Controller shouldn't have access to db layer. It should only know about http layer. Let's create something like `Services` or `Handlers` and call db queries there.